common/changes/@bentley/bentleyjs-core/api-promotions_2021-05-19-16-52.json

```diff
@@ -2,10 +2,14 @@
   "changes": [
     {
       "packageName": "@bentley/bentleyjs-core",
+<<<<<<< HEAD:common/changes/@bentley/bentleyjs-core/api-promotions_2021-05-19-16-52.json
       "comment": "Promote APIs to public.",
+=======
```

Comment:
here too

Reply:
plenty more; grepping and fixing...

Reply:
btw why are packages like rpc-full-stack-tests producing changelogs? Is that opt-out somewhere?

Reply:
fixed.

Reply:
The rpc-full-stack-tests are being published as a package so they include changelogs as well.